packages/atomic/src/components/atomic-modal/atomic-modal.pcss

```diff
@@ -30,53 +11,71 @@ atomic-focus-trap {
 [part='container'] {
   @apply overflow-hidden;
   grid-area: modal;
+}
 
-  @screen mobile-only {
-    @apply absolute inset-0;
-    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.1);
+:host(.open) {
+  [part='backdrop'] {
+    @apply pointer-events-auto;
   }
+}
 
-  @screen desktop-only {
-    @apply rounded;
-    min-width: 30rem;
-    max-width: 100vw;
-    max-height: 100vh;
+:host(.open.dialog) {
+  [part='backdrop'] {
+    background-color: rgba(40, 40, 40, 0.8);
   }
 }
 
-[part='header-wrapper'] {
-  @screen mobile-only {
-    @apply p-6;
+:host(.dialog) {
+  [part='backdrop'] {
+    @apply grid p-6;
+    transition: background-color 500ms ease-in-out;
+
+    grid-template-areas:
+      '. .     .'
+      '. modal .'
+      '. .     .';
+
+    grid-template-columns: 1fr min(30rem, 100%) 1fr;
+    grid-template-rows: 1fr auto 3fr;
+  }
+
+  [part='container'] {
+    @apply rounded;
   }
 
-  @screen desktop-only {
+  [part='header-wrapper'] {
     @apply px-6 py-4;
   }
-}
 
-[part='header'] {
-  @screen desktop-only {
+  [part='header'] {
     @apply font-bold;
   }
+
+  [part='body-wrapper'] {
+    @apply p-6;
+  }
+
+  [part='footer-wrapper'] {
+    @apply bg-neutral-light;
+    padding: 1rem 1.125rem;
+  }
 }
 
-[part='body-wrapper'] {
-  @screen mobile-only {
-    @apply px-6 pt-8 pb-5;
+:host(.fullscreen) {
+  [part='container'] {
+    @apply absolute inset-0;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.1);
```

Comment:
nothing in tailwind for this?

Reply:
not really, I can define them in the theme but we're not re-using it and it seems pretty specific to this use case, so I don't see a good reason to do that yet.

Reply:
I'm not sure I understand why we can't just add the class `shadow` or `shadow-md` on the container ?

What do you need to define exactly ? The exact values from the mockup ?

Because if that is so, it's much better to just use tailwind default values, even if it's a bit different than the mockups.

Reply:
Good point, I'll try a tailwind value instead.